src/js/components/Hotspot.js

```diff
@@ -0,0 +1,20 @@
+import React, { PropTypes } from 'react';
+import Add from './icons/base/Add';
+import CSSClassnames from '../utils/CSSClassnames';
+
+const CLASS_ROOT = CSSClassnames.HOTSPOT;
+
+export default function Hotspot (props) {
+  return (
+    <div className={CLASS_ROOT} onClick={props.onClick}>
```

Comment:
Would it be possible to separate the visual aspects from the interaction aspects such that HotSpot could be just an icon that we could place inside a Button or Anchor? That would allow us to get all of the accessibility from Button and Anchor but the visual styling from HotSpot.